locale/en/index.md

```diff
@@ -13,6 +13,9 @@ labels:
   tagline-lts: Mature and Dependable
   changelog: Changelog
   api: API Docs
+  version-schedule-prompt: Or have a look at the
+  version-schedule-prompt-link: https://raw.githubusercontent.com/nodejs/LTS/master/schedule.png
```

Comment:
This URL doesn't need to be different in other languages, right? If so it could just be placed directly inside the href-attribute in `layouts/index.hbs`